browser_use/browser/browser.py

```diff
@@ -259,6 +274,7 @@ async def _setup_builtin_browser(self, playwright: Playwright) -> PlaywrightBrow
 			offset_x, offset_y = get_window_adjustments()
 
 		chrome_args = {
+			CHROME_DEBUGGING_ARG % {'chrome_remote_debugging_port': self.config.chrome_remote_debugging_port},
```

Comment:
```suggestion
			f'--remote-debugging-port={self.config.chrome_remote_debugging_port}',
```

Just hardcode it no need for the imported constant imo